ccdproc/tests/test_ccdproc.py

```diff
@@ -531,3 +531,10 @@ def test__blkavg_larger(ccd_data):
 
     assert b.shape == (10, 10)
     np.testing.assert_almost_equal(b.sum(), 0.25 * a.sum())
+
+#test overscan changes
+def test__overscan_schange(ccd_data):
+    old_data=ccd_data.copy()
```

Comment:
Please put single space around equals sign, after commas, etc. When you have a chance take a look at [PEP 8](https://www.python.org/dev/peps/pep-0008/)